.gitignore

```diff
@@ -5,6 +5,7 @@
 *.suo
 *.user
 *.sln.docstates
+.idea/
```

Comment:
Not specifically, but for some reason Webstorm decided to add that folder my project.